### DISCUSSION TITLE
Long sync. Is mainnet/db/blockchain_v1_mainnet.sqlite available for download?

### DISCUSSION THREAD

**Discussion by madchiaman777** [Chia-Network/chia-blockchain#7137] [Ideas]
Hello!
Synchronization takes several days.
My Windows client doesn't start with db from Linux.
Is it possible to get up-to-date blockchain directory?
Is it safe if I get DB from unknown source?

**Comment by loppefaaret** [comment]
no, it is not safe to get a DB from a 3rd party!
